src/qmapshack/gis/CGisWorkspace.cpp

```diff
@@ -923,7 +923,7 @@ void CGisWorkspace::addWptByPos(const QPointF& pt, const QString& name, const QS
     CGisItemWpt::newWpt(pt, name, desc, project);
 }
 
-void CGisWorkspace::addPoisAsWpt(const QList<poi_t> &pois, IGisProject * project) const
+void CGisWorkspace::addPoisAsWpt(const QSet<poi_t> &pois, IGisProject * project) const
```

Comment:
I mean either way is ok for me but it should be consistent over the code, e.g. you will (should) not find WPT anywhere. It's always Wpt and so on.

Reply:
`IMap` and `CMapIMG` uses "POI", should I adjust the code to use "POI" everywhere? Should `IPoi` then also be renamed to `IPOI` and `CPoiPOI`to `CPOIPOI` to follow the naming convention used for the maps?

Reply:
The coding guideline states:

> Use camel case with first character lowercase on variables.

Therefore `POI`, `POIS` should always be written `Poi`, `Pois`. When I wrote the code ages ago I decided to write file extensions for maps and DEM in capital letters. Like `CMapIMG` or `CDemVRT`. That is the reason for CPoiPOI. This is the class derived from `IPoi` that handles files with extension `*.poi` 

Therefore writing `Poi`/`Pois` for all API methods is the logical consequence. I am not sure if I really kept up with that scheme in the complete code :sweat_smile: As said I am not *that* strict about it. But everything concerning POIs should be written the same way. By that I can easily guess an API name or search for it in the code. And for those calls that go over several classes the name should be always the same. By that you search for the name and know all classes that are affected by that API. Additionally they should follow the same calling interface if possible.

Don't pin me on that because I am sure that you will find violations of that scheme in the code. Usually I start to fix that if I notice. Good portions of that code have been written during late hours with the usual consequences :wink: